.github/workflows/pull-request.yml

```diff
@@ -9,8 +11,8 @@ jobs:
       - name: "Checkout Branch"
         uses: actions/checkout@v2
         with:
-          repository: ${{github.event.pull_request.head.repo.full_name}}
-          ref: ${{github.head_ref}}
+          repository: ${{ github.event.pull_request.head.repo.full_name }}
+          ref: ${{ github.head_ref }}
```

Comment:
nit: we can remove the repository/ref fields since we are no longer using `pull_request_target` :)